pageserver/src/billing_metrics.rs

```diff
@@ -58,6 +66,7 @@ impl BillingMetric {
             tenant_id,
             timeline_id,
             time: Utc::now(),
+            idempotency_key: format!("{}", Utc::now()),
```

Comment:
can we also add a random integer, please, so that the result was "{time}-{hostname_or_nodeid}-{random-4-bytes-number}". Because we need the idempotency_key to be unique across events sent in 38 hours range and if Pageserver sends 2 events at a time, I assume `format!("{}", Utc::now())` called twice for two events can produce the same string.

Reply:
Fixed. 
New format is
```
{
'metric': 'remote_storage_size', '
 type': 'absolute', 
 'tenant_id': 'a03ebb4f5922a1c56ff7485cc8854143',
 'time': '2022-12-28T11:07:19.317310284Z',
 'idempotency_key': '2022-12-28 11:07:19.317310324 UTC-1-4019', 
'value': 25600000
}
```